src/appleseed/foundation/utility/stopwatch.h

```diff
@@ -129,12 +143,37 @@ template <typename Timer>
 inline void Stopwatch<Timer>::clear()
 {
     m_elapsed = 0;
+    m_commited = 0;
 }
 
 template <typename Timer>
 inline void Stopwatch<Timer>::start()
 {
+    m_paused = false;
```

Comment:
From your description, `start()` cannot be called in pause state. This should be an assert.

Reply:
Actually, I would prefer updating my description because it's really convenient to call `start` to restart everything

Reply:
Ok.